pkg/uroot/uroot.go

```diff
@@ -160,6 +166,20 @@ func CreateInitramfs(opts Opts) error {
 		}
 	}
 
+	if opts.ExtraDir != "" {
```

Comment:
extrafiles should already cover this use case. Why is this necessary?

Reply:
A couple lazy reasons: 1) the LDD routine was failing on including the musl flashrom in Heads, 2) I couldn't figure out how to get all files in a directory included in the archive relative to the root (e.g. all files in "/tmp/uinitrd-root/{etc,dev}/" included at "/{etc,dev}/".  This seemed like the cheapest / widely usable option.

 I suppose I could use the base cpio feature instead.  Am I missing something obvious?

Reply:
so, let's try to fix our tools if we can rather than providing workarounds in our tools for problems in our other tools :-)
You mention two issues here. Can you give us commandline examples so we can fix them?

Reply:
done